common/k8stest/util_apps.go

```diff
@@ -96,6 +97,157 @@ func (dfa *FioApp) validate() error {
 	return nil
 }
 
+func (dfa *FioApp) Deployment(replicaCount int) error {
+	err := dfa.validate()
```

Comment:
I see code are getting repeated from `DeployFio` method . Can you use `DeployFio` and create deployment not pod in case of deployment ?

Reply:
Modified as suggested